src/Microsoft.AspNetCore.DataProtection/DataProtectionConfiguration.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.AspNetCore.DataProtection
     /// <para>
     /// If the developer changes the at-rest key protection mechanism, it is intended that
     /// he also change the key storage location, and vice versa. For instance, a call to
-    /// <see cref="ProtectKeysWithCertificate(string)"/> should generally be accompanied by
+    /// <c>ProtectKeysWithCertificate(string)</c> should generally be accompanied by
```

Comment:
ifdef the comments too so it's right on whatever platform? It needs to be there in supported platforms